crates/cheatnet/src/forking/state.rs

```diff
@@ -177,10 +177,10 @@ impl StateReader for ForkStateReader {
             if let Some(cache_hit) = self.cache.get_compiled_contract_class(class_hash) {
                 Ok(cache_hit)
             } else {
-                match self.runtime.block_on(
-                    self.client
-                        .get_class(self.block_id, FieldElement::from_(*class_hash)),
-                ) {
+                match self.runtime.block_on(self.client.get_class(
+                    BlockId::Number(self.block_number.0),
```

Comment:
Would it be possible to create a function/ other suitable solution for this line? It is repeated few times

Reply:
Will change it to a function, just didn't want to have `self.block_id` since it allows other variants than `BlockId::Number`